src/server/services/wfs/qgswfsparameters.h

```diff
@@ -56,8 +58,29 @@ namespace QgsWfs
         EXP_FILTER,
         GEOMETRYNAME
       };
-      Q_ENUM( ParameterName )
+      Q_ENUM( Name )
 
+      QgsWfsParameter( const QgsWfsParameter::Name name = QgsWfsParameter::UNKNOWN,
+                       const QVariant::Type type = QVariant::String,
+                       const QVariant defaultValue = QVariant( "" ) );
+
+      int toInt() const;
+      QStringList toStringListWithExp( const QString &exp = "\\(([^()]+)\\)" ) const;
```

Comment:
You may want to use a raw string for readability

Reply:
Thanks for the review @elpaso!

Can you give me an example of what you mean?

Reply:
See this article for an explanation: http://www.informit.com/articles/article.aspx?p=2064649&seqNum=2 , and https://en.cppreference.com/w/cpp/language/string_literal
Basically you don't need to escape the backslashes. But it's really a minor thing: feel free to ignore my comment.